### PR TITLE
opera => 74.0.3911.154

### PR DESCRIPTION
**Pull request by supechicken**
Tested on `x86_64`

### CODE REVIEW

**Comment by satmandu** [comment]
Does this work? An upgrade of opera failed for me this evening.

**Comment by uberhacker** [comment]
We're ready for another update anyway.

File: packages/opera.rb [hunk 1]
```diff
@@ -3,34 +3,35 @@
...
 
   def self.build
-    system "curl https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb -#o opera.deb"
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera.deb") ) == `curl -Ls https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`
-    puts "Archive downloaded".lightgreen
-    system "alien -tc opera.deb"
-    system "tar xvf opera-stable-#{version}.tgz > /dev/null"
+    Dir.chdir('../') do
+      system "alien -tc opera-stable_#{version}_amd64.deb"
+      system "tar xf opera-stable-#{version}.tgz"
```
Comment (uberhacker) [inline packages/opera.rb:24]: We don't need to do this anymore, right?

**Comment by supechicken** [comment]
> Does this work? An upgrade of opera failed for me this evening.

It works before but seems #5247 broke the unarchiving process
We may need to update all deb packages for it

File: packages/opera.rb (hunk 1 above)
Comment (supechicken) [reply packages/opera.rb:24]: A pr for all deb packages is coming soon:)

**Pull request merged**